src/api/firebase.js

```diff
@@ -11,7 +11,9 @@ import {
 } from 'firebase/firestore';
 import { useEffect, useState } from 'react';
 import { db } from './config';
-import { getFutureDate } from '../utils';
+import { getFutureDate, getDaysBetweenDates } from '../utils';
+import { calculateEstimate } from '@the-collab-lab/shopping-list-utils';
+// calculateEstimate takes 3 arguments - previousEstimate (number), daysSinceLastPurchase (number), totalPurchases (number)
```

Comment:
We left it in because we have no other in app reference for how calculateEstimate works, and we thought it could be useful if we ever needed to make changes to this part of the code.

Reply:
That makes sense! Having it there will be a lot easier to reference than having to go digging.